src/ucp/core/ucp_ep.inl

```diff
@@ -17,27 +17,58 @@
 #include <ucs/arch/bitops.h>
 #include <ucs/datastruct/ptr_map.inl>
 
+#define UCP_EP_FAST_LANE_MASK (UCS_BIT(UCP_LANE_TYPE_AM) | \
+                               UCS_BIT(UCP_LANE_TYPE_RMA) | \
+                               UCS_BIT(UCP_LANE_TYPE_AMO) | \
+                               UCS_BIT(UCP_LANE_TYPE_CM) | \
+                               UCS_BIT(UCP_LANE_TYPE_TAG))
 
 static inline ucp_ep_config_t *ucp_ep_config(ucp_ep_h ep)
 {
     ucs_assert(ep->cfg_index != UCP_WORKER_CFG_INDEX_NULL);
     return &ep->worker->ep_config[ep->cfg_index];
 }
 
+static UCS_F_ALWAYS_INLINE uct_ep_h ucp_ep_get_fast_lane(ucp_ep_h ep,
+                                                         ucp_lane_index_t lane_index)
```

Comment:
line too long

Reply:
inl files ignored by clang--format, I'll check it out